viewer/templates/viewer/help.html

```diff
@@ -6,17 +6,23 @@
   </a>
 </nav>
 
-<h2 class="o-header_alt">Common web page index searches</h2>
+<h2 class="o-header_alt">Common Searches</h2>
 
 <p>
-  The web page index stores a nightly snapshot of webpage data that you can
-  search for all sorts of useful things. You can search for single terms or
-  exact phrases.
+  This tool stores a nightly snapshot of consumerfinance.gov web page data that
+  you can search for all sorts of useful things. Common search examples are
+  below. You can also
+  <a href="">learn more about the data</a>. View results in the browser, or
+  export them as a CSV. For those less handy with Excel, it may be useful to
+  <a
+    href="https://github.cfpb.gov/CFPB/el-camino/issues/642#issuecomment-318309"
```

Comment:
We shouldn't link to GHE from public GitHub. Shouldn't this link directly to the Word doc as a download? I think we will need to add this file to the repo so it can be downloaded.

Reply:
LOL, I am dumb. Clearly not firing on all cylinders today 😂 
Thanks for catching this, yes we need to add the file to the repo.